amo2kinto/synchronize.py

```diff
@@ -37,8 +61,10 @@ def push_changes(diff, kinto_client, bucket, collection):
             # Records are enabled by default.
             record['enabled'] = True
             batch.create_record(record)
+        for record in to_update:
+            batch.patch_record(strip_keys(record))
```

Comment:
To start with there are no such things as enabled in the AMO record, are you suggesting that we force it back to true there? Same for last_modified, we don't have it. Here it is especially to strip the ID but I can specify the keys argument.